### DISCUSSION TITLE
You already have nginx. How do you configure nginx when you install documentserver?

### DISCUSSION THREAD

**Discussion by itmeicn** [ONLYOFFICE/DocumentServer#1788] [Q&A]
You already have nginx. How do you configure nginx when you install documentserver? Is there a tutorial?

**Comment by ShockwaveNN** [comment]
Sorry, I don't think we got a tutorial for that, you can try to copy-paste the config from here, but you need to know that are you doing

If you already got nginx and do not want to reconfigure it - I think `docker` version of DocumentServer will be a better variant for you

**Comment by jiriks74** [comment]
@gaohc001 I solved this by running DocumentServer on another port. There's a command to change it before the installation. That's the fastest and easiest.

**Reply by itmeicn** [reply]
I have old nginx How to configure onlyoffice?please give *.conf for  me

**Reply by itmeicn** [reply]
My solution, other web userd apache.only documentserver used nginx .

**Reply by ShockwaveNN** [reply]
> I have old nginx How to configure onlyoffice?please give *.conf for me

Sources located [here](https://github.com/ONLYOFFICE/document-server-package/tree/master/common/documentserver/nginx)
